src/Toast/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Omit } from '@stoplight/types';
+import * as React from 'react';
+import {
+  Bounce as ReactTransitionBounce,
+  Flip as ReactTransitionFlip,
+  Slide as ReactTransitionSlide,
+  toast as ReactToast,
+  ToastOptions,
+  TypeOptions as ToastType,
+  Zoom as ReactTransitionZoom,
+} from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+import { IBox } from '../Box';
+import { IToastContent, ToastContent } from './ToastContent';
+
+/**
+ * TOAST
+ */
+export interface IToast extends IToastContent, Omit<ToastOptions, 'transition'> {
+  transition?: IToastTransition;
+  attributes?: IBox<HTMLElement>;
+}
+
+export function Toast(options: IToast) {
+  const {
+    title,
+    message,
+    icon,
+    closeIcon,
+    render: customRenderer,
+    attributes = {},
+    transition = 'zoom',
+    ...toastOptions
```

Comment:
using attributes as the box properties that get passed to `ToastContent` so we a can use the spread operator for `toastOptions`. Doing this because Toast is a function and not a component so it felt dirty to pass the component props directly? Also so we can spread `toastOption`